.agents/skills/triage-issues/evals/eval_runner.py

```
@@ -0,0 +1,384 @@
+    """
+    # Parse roadmap cited issues
+    in_progress_issues = set(re.findall(r"\(#(\d+)\)", re.search(r"## ⏳ In Progress(.*?)(##|\Z)", roadmap_content, re.S).group(1))) if "## ⏳ In Progress" in roadmap_content else set()
+    planned_issues = set(re.findall(r"\(#(\d+)\)", re.search(r"## 📅 Planned(.*?)(##|\Z)", roadmap_content, re.S).group(1))) if "## 📅 Planned" in roadmap_content else set()
+
+    results = {}
+
+    for issue in open_issues:
+        num = str(issue.get("number"))
+        title = issue.get("title", "")
+        body = issue.get("body", "")
+        labels = [l.get("name") for l in issue.get("labels", [])]
```

Ruff (0.15.20)

[error] Ambiguous variable name: `l`

(E741)